src/common/low_precision_transformations/include/low_precision/layer_transformation.hpp

```diff
@@ -273,6 +273,9 @@ class LP_TRANSFORMATIONS_API LayerTransformation : public ov::pass::MatcherPass
         }
 
         bool updatePrecisions;
+        // Use deqPrecision only for FakeQuantize operation decomposition,
+        // use existing precisions in other cases.
+        // In this case if FakeQuantize operations were decomposed then original precision will be used.
```

Comment:
This comment is a bit confusing, because there are a lot of `deqPrecision` usages in LPT besides FQ decomposition (e.g. `AddTransformation`/`ConvolutionTransformation`/`MatMulTransformation`). Should we avoid `deqPrecision` usage there as well?

Reply:
Ideally yes, but not every time and not right now. It affects quantized mixed precision models (in preactice: fp16 Q/DQ models: GPU, NPU, CPU ARM) only. Let's disccuss online.